Sources/Fuzzilli/Core/Corpus.swift

```diff
@@ -38,6 +38,8 @@ public class Corpus: ComponentBase {
     /// The current set of interesting programs used for mutations.
     private var programs: RingBuffer<Program>
     private var ages: RingBuffer<Int>
+
+    private var typeExtensionDeduplicationSet = WeakSet<TypeExtension>()
```

Comment:
Maybe add a comment here that the Corpus deduplicates the runtime types of its programs to conserve memory.